test/test262/test262.js

```diff
@@ -119,7 +168,34 @@ if (!process.send) {
   };
 
   (async () => {
-    const files = override ? glob.sync(path.join(TEST262_TESTS, override)) : readdir(TEST262_TESTS);
+    let files = [];
+    if (ARGS.length === 0) {
+      files = readdir(TEST262_TESTS);
+    } else {
+      // Interpret pattern arguments relative to the tests directory,
+      // falling back on the working directory if there are no matches
+      // or a non-glob pattern fails to match.
+      for (const arg of ARGS) {
+        const matches = globby.sync(arg, { cwd: TEST262_TESTS, absolute: true });
+        if (matches.length === 0 && !globby.hasMagic(arg)) {
+          files = [];
+          break;
+        }
+        files.push(...matches);
+      }
+      if (files.length === 0) {
+        const cwd = process.cwd();
+        for (const arg of ARGS) {
+          const matches = globby.sync(arg, { cwd, absolute: true });
+          if (matches.length === 0 && !globby.hasMagic(arg)) {
+            fs.accessSync(path.resolve(cwd, arg), fs.constants.R_OK);
```

Comment:
Exceptions raised by this `accessSync` avoid silent skipping of fat-fingered patterns like `test/built-ins/BigInt` (which does not exist because `test` is implied).